src/main/java/mybatis/com/mybatis/Dto/StudentDtoForSubject.java

```diff
@@ -0,0 +1,13 @@
+package mybatis.com.mybatis.Dto;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class StudentDtoForSubject {
+    private Integer id;
+    private String fullName;
+    private List<SubjectDto> subjectDtoList;
```

Comment:
```suggestion
    private List<SubjectDto> subjects;
```

Response should never include terms like DTO, entity etc

Reply:
done